interfaces/acados_matlab/ocp_compile_mex.m

```diff
@@ -58,16 +62,22 @@ function ocp_compile_mex(opts)
 		end
 	else
 		if (strcmp(opts.qp_solver, 'full_condensing_qpoases'))
-			mex(mex_flags, 'CFLAGS=\$CFLAGS -std=c99 -fopenmp -DACADOS_WITH_QPOASES', acados_include, acados_interfaces_include, external_include, blasfeo_include, acados_lib_path, '-lacados', '-lhpipm', '-lblasfeo', '-lqpOASES_e', mex_files{ii})
```

Comment:
why removing `\`?

Reply:
Fails on windows with the message:
```
gcc: error: \-fexceptions: No such
file or directory
```
so the `\` seems to end up in the gcc command.

I do not understand the meaning of the `\` (escape?) here. Why is it there? The mex command works without on Win/Linux/Matlab/Octave, but i am not sure if potentially CFLAGS are getting lost.

Reply:
Ok, we need to check on older matlab versions I guess. I found that on online forums and it works on my machines.

Reply:
so in the docs it is described and they are not using the `\`:
```
[mex docs - Override Default Compiler Switch Option](https://www.mathworks.com/help/matlab/ref/mex.html#btw193g-1)
```
but they only show examples for the command line form, and the functional form of `mex`.
On google i found both usages, but no explanation, so i guess we need to test it ourself.